README.md

```diff
@@ -143,25 +233,27 @@ We run as much as we can on Heroku, because that environment puts devs in direct
 
 The remainder of our environment is hosted in AWS. We make heavy use of EC2, RDS, EBS, and S3.
 
-### Testing & monitoring
+### Testing & Monitoring
 
 We maintain quality with unit and functional testing, and we’re constantly trying to find better ways to test. We strongly prefer frameworks, like Laravel, that have unit testing built in to their core.
 
-We monitor with New Relic, StatHat, Runscope, and Ghost Inspector. We centralize alerts and alerting policies in PagerDuty.
+We [monitor](https://github.com/DoSomething/communal-docs/tree/master/Monitoring) with New Relic, StatHat, [Runscope, and Ghost Inspector](https://github.com/DoSomething/communal-docs/tree/master/Automated%20Tests). We centralize alerts and alerting policies in PagerDuty. We have an [Incident Playbook](https://github.com/DoSomething/communal-docs/tree/master/Incidents) and we document common issues we might see for a given service.
+
+Every Wednesday of the month we hold a Post-mortem / Incident review meeting. Tech leads are responsible for creating the tickets generated from the meeting (if they don't exist already) and following up with Product Managers to discuss prioritization. These tickets are placed in the Collective Freezer Pivotal Tracker board. At the start of the meeeting, we review tickets, followup items from the previous month, and discuss the incidents that have occurred.
 
 ### Documentation
 
-We believe in great documentation—not for documentation sake, but to make on-boarding and institutional memory better. Writing good docs also makes you more thoughtful about your choices. It's like teaching: you can't teach something you haven't thought through all the way.
+We believe in great documentation — not for documentation sake, but to make on-boarding and institutional memory better. Writing good docs also makes you more thoughtful about your choices. It's like teaching: you can't teach something you haven't thought through all the way.
 
 We document in code \(doc blocks, etc.\), and in Markdown Readme files in the source repos. Sometimes the project-level Readme is sufficient, but for [bigger projects](https://github.com/DoSomething/northstar/tree/dev/documentation) we'll break the docs into their own directory.
 
-### Data collection
+### Data Collection
 
 We collect raw data in our [Quasar](https://github.com/DoSomething/quasar) warehouse; we prefer making apps push their events to the warehouse rather than writing ETL tools to scrape APIs.
```

Comment:
Maybe here we could change to something like: 
"We collect raw data in our warehouse and transform data in DBT; we prefer making apps push their events to the warehouse rather than writing ETL tools to scrape APIs and we prefer a layered (hierarchical) data model approach to a flat design to ensure endpoints are comprehensible by end users."

Reply:
Updated! (I linked "our warehouse" to the Quasar repo